tutorials/napps/development_environment_setup.rst

```diff
@@ -138,29 +138,36 @@ Cloning existing projects
 If you want to contribute with a Kytos project by coding, you must clone/fork
 its repository located at `GitHub group <https://github.com/kytos>`_. Below, we
 are going to clone all major repositories (python-openflow, kytos-utils
-kytos-napps and kytos):
+and kytos):
 
 .. code-block:: bash
 
-  $ for project in python-openflow kytos-utils kytos kytos-napps; do
+  $ for project in python-openflow kytos-utils kytos; do
```

Comment:
Shall we remove kytos-napps from the dev_env?
As we are installing the projects in develop mode, having the NApps right and ready would be nice.

If we are going to remove it, there will be changes in the logs of `kytos napps list` all over the tutorials (since no NApp will be installed at that point), which need updating. There are also text instructions, like in "How to use Kytos with Mininet", that say "You probably already have them (our NApps)", and adjusting them would be nice.